assets/cluster-monitoring-operator/service-monitor.yaml

```diff
@@ -7,11 +7,19 @@ metadata:
   namespace: openshift-monitoring
 spec:
   endpoints:
-  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+  - bearerTokenFile: ""
+    metricRelabelings:
+    - action: drop
```

Comment:
why these metrics are dropped?

Reply:
The [jsonnet code](https://github.com/openshift/cluster-monitoring-operator/pull/1870/files#diff-a89d64ee6e92f3dd1e7faf2c0e4bdf29ea537f3a82d7dd8db62490a0e8799efc) has the answer :)